mgmt/rest/v2/metric.go

```diff
@@ -0,0 +1,92 @@
+package v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+
+	"github.com/codegangsta/negroni"
+	"github.com/intelsdi-x/snap/core"
+	"github.com/intelsdi-x/snap/mgmt/rest/rbody"
+	"github.com/intelsdi-x/snap/mgmt/rest/rbody/v2"
+)
+
+func RespondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter) {
+	b := v2.NewMetricsReturned()
+
+	for _, met := range mets {
+		rt := met.Policy().RulesAsTable()
+		policies := make([]rbody.PolicyTable, 0, len(rt))
+		for _, r := range rt {
```

Comment:
I feel there is a duplicate of code with https://github.com/intelsdi-x/snap/blob/67417ef0fca51def9cf0eaf002d0dc42c0aee842/control/plugin/cpolicy/node.go#L131-L147

It's better to:
- Create a method `RulesAsPolicyTable`.
- Or use the type `cpolicy.RuleTable` instead of `rbody.PolicyTable`, those type are exactly the same at 2 different places.